draft-ietf-scone-protocol.md

```diff
@@ -300,50 +305,45 @@ header cannot precede any other packets.
 
 ## Rate Signals {#rate-signal}
 
-The Rate Signal field in SCONE uses the low 6 bits (0x3f) of the first byte.
-This field is encoded as a logarithmically spaced distribution over a range
-defined by the SCONE protocol version.
+The Rate Signal field in SCONE uses 7 bits: the low 6 bits (0x3f) of the first
+byte combined with the most significant bit (0x80) of the second byte.
+This creates a contiguous 7-bit field with values from 0 to 127.
 
```

Comment:
Same comment. If we end up calling the 6 bits Rate _Signal High Bits (6)_ in the diagrams, lets reuse the name here.

Reply:
Fixed in latest commit.